.gitmodules

```diff
@@ -193,3 +193,6 @@
 [submodule "contrib/miniselect"]
 	path = contrib/miniselect
 	url = https://github.com/danlark1/miniselect
+[submodule "contrib/xz"]
+       path = contrib/xz
```

Comment:
Uses spaces for indent, while all other lines uses tabs (since `git submodule add` uses them)